capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -36,7 +34,7 @@ export class SearchUserModal extends Component {
     else {
       searchResults = await fetch(`/api/userSearch?searchTerm=${searchTerm}`)
         .then(response => response.json())
-        .catch(err => alert(err));
+        .catch(err => console.log(err));
```

Comment:
nit: `console.error` would probably be better here.

Reply:
Ok.